<launch>
  <!-- "camera" should uniquely identify the device. All topics are pushed down
       into the "camera" namespace, and it is prepended to tf frame ids. -->
  <arg name="camera" default="XTION" />
  <arg name="object_mesh"  default="wheel.obj" />
  <arg name="depth_frame" default="/$(arg camera)_IR" />
  <arg name="calibration_url"  default="$(find state_filtering)/config/xtion_calib.yaml" />
  <arg name="noise"  default="NONE" />
  <node pkg="render_kinect" name="$(arg camera)" type="move_object" respawn="false" output="screen">
    <param name="object_mesh"  type="string"  value="$(arg object_mesh)" />
    <param name="depth_frame_id"  type="string"  value="$(arg depth_frame)" />
    <param name="calib_url"  type="string"  value="$(arg calibration_url)" />
    <param name="noise"  type="string"  value="$(arg noise)" />
  </node>
  
  
  <!-- Load reasonable defaults for the relative pose between cameras -->
  <include file="$(find render_kinect)/launch/kinect_frames.launch">
    <param name="camera" type="string" value="/$(arg camera)"/>
  </include>
</launch>